models/iaf_psc_exp_ps_lossless.h

```diff
@@ -268,7 +268,7 @@ class iaf_psc_exp_ps_lossless : public ArchivingNode
    * threshold line V < \theta, envelope, V > b(I_e) and line corresponding to
    * the final timestep
    * V > f(h, I) (or) linear approximation of the envelope, V < g(h, I_e).
-   * Note that in Algorithm 1 and 2 of [1], a typo interchanges g and f.
+   * Note that in Algorithm 1 and 2 of [1]_, a typo interchanges g and f.
```

Comment:
No `:math:` in the equations above?

Reply:
This is not part of the userdocs section, so the underscore is actually incorrect. Removing it.